src/HexagonOffload.cpp

```diff
@@ -67,18 +67,19 @@ class InjectHexagonRpc : public IRMutator {
     Scope<ModulusRemainder> alignment_info;
 
     Expr state_var(const std::string& name, Type type) {
-        Expr& var = state_vars[name];
-        if (!var.defined()) {
-            auto storage = Buffer<void *>::make_scalar(name + "_buf");
-            storage() = nullptr;
-            var = Load::make(type_of<void*>(), storage.name(), 0, storage, Parameter(), const_true());
-        }
-        return var;
+        return Let::make(name, state_var_ptr(name, type),
```

Comment:
Here is an example of loading directly from a symbol of type Handle, instead of requiring an Allocate node in scope.